### PR TITLE
Multi lang tabs

### PR DESCRIPTION
**Pull request by roeierez**
This PR adds the multi language tab for code examples.
@yaslama has implemented that and I added the swift examples.

### CODE REVIEW

**Comment by dangeross** [comment]
I'll add RN and Go examples.

File: src/guide/getting_started.md [hunk 1]
```diff
@@ -62,4 +67,72 @@ if let Some(node_state) = sdk.node_info()? {
...
 ```
+</section>
+<div slot="title">Swift</div>
+<section>
+
+The first step is to register a new node
+## Registering a new node
+```swift
+do {
+ let seed = try mnemonicToSeed(phrase: "<mnemonics words>");
```
Comment (ubbabeck) [inline src/guide/getting_started.md:78]: NIT: Adding a semicolon at the end of lines in Swift is optional and generally not preferred. Think we can remove them. 
https://github.com/kodecocodes/swift-style-guide#semicolons

**Review by ubbabeck — APPROVED** [review]
NIT, see comment about semicolons. 

Looks good to me.

**Review by dangeross — APPROVED** [review]
I'll add Go examples in a separate PR as the Go package needs to be built off the next SDK release.

**Review by erdemyerebasmaz — APPROVED** [review]
Before we add more changes on this PR, let's merge these changes so we have a visible example* on https://sdk-doc.breez.technology/ and it would also be a better reference if each language support has their own PR.

*: There's an in-built preview tool in VSCode if you'd like to see your changes while adding new language supports.

**Pull request merged**